command-line/command-line-interface.md

```diff
@@ -26,6 +26,34 @@ $ cd Downloads/
 - Go to www.npmjs.com/package/express, which is the Node Package Manager page for Express.js.
 - Everything in the block after the "$" is the command that you would type into the Command Line in order to install Express.js.
 
+### Set Up zsh
+First, if you purchased a Mac with the latest operating system, **Catalina**, *zsh* is already the default shell. Be sure to check by typing the following to your command line:
+
+```
+zsh --version
+```
+
+If you currently own a Mac and just recently upgraded your operating system to Catalina, please type the following line to switch the default shell from *Bash* to *zsh*:
```

Comment:
A lot of apprentices have older, donated laptops that may not run the latest Mac. Can you add a section to help them install zsh? I believe I just did `brew install zsh`.

Reply:
Thank you. Should I also include how to install Homebrew also in case some Macs don't have Homebrew installed?